pkg/kudoctl/cmd/get/get_test.go

```diff
@@ -57,53 +55,18 @@ func TestGetInstances(t *testing.T) {
 		},
 	}
 	tests := []struct {
-		arg       []string
-		err       string
 		instances []string
 	}{
-		{nil, "expecting exactly one argument - \"instances\"", nil},                                   // 1
-		{[]string{"arg", "arg2"}, "expecting exactly one argument - \"instances\"", nil},               // 2
-		{[]string{}, "expecting exactly one argument - \"instances\"", nil},                            // 3
-		{[]string{"somethingelse"}, "expecting \"instances\" and not \"somethingelse\"", nil},          // 4
-		{[]string{"instances"}, "expecting \"instances\" and not \"somethingelse\"", []string{"test"}}, // 5
+		{[]string{"test"}},
```

Comment:
All of these test cases already exist above in `TestValidate` and were effectively a no-op here.